libs/portals/shared-modules/delegations/src/components/Modal/Modal.tsx

```diff
@@ -30,47 +49,39 @@ export const Modal = ({
     }
   }
 
+  const { formatMessage } = useLocale()
+  const { md } = useBreakpoint()
+
   return (
     <ModalBase
       baseId={id}
-      isVisible={isVisible}
-      className={styles.modal}
       modalLabel={label}
+      className={styles.modal({ noPaddingBottom })}
+      isVisible={isVisible}
+      onVisibilityChange={handleOnVisibilityChange}
       hideOnClickOutside
       hideOnEsc
-      onVisibilityChange={handleOnVisibilityChange}
+      preventBodyScroll={false} // Do to a bug in iOS we implement our own scroll lock
     >
-      <Box
-        position="relative"
-        background="white"
-        paddingX={[3, 3, 6]}
-        paddingBottom={noPaddingBottom ? 0 : [3, 3, 6]}
-        paddingTop={12}
-        borderRadius="standard"
-        width="full"
-        className={styles.modalInner}
-      >
-        <Box
-          display="flex"
-          flexDirection="column"
-          rowGap={2}
-          tabIndex={-1}
-          ref={headingRef}
-        >
-          {label && (
-            <Text variant="small" fontWeight="semiBold">
-              {label}
+      <Box className={styles.header} ref={headingRef}>
+        <Box display="flex" flexDirection="column" rowGap={1}>
+          {eyebrow ? <Text variant="eyebrow">{eyebrow}</Text> : null}
```

Comment:
I always try to use ternaries rather than Short-circuiting with && because if the value is a falsy, that value will be rendered. This is not a big problem, but I thater be precise in my logic, even if it is a bit uglier.

I have had cases where a 0 is rendered to the DOM.